### PR TITLE
Added some options and verbosity to ari_spin

### PR DESCRIPTION
**Pull request by muschellij2**
Added verbose argument in case you want to see what is happening with the commands behind the scenes.

### CODE REVIEW

**Comment by seankross** [comment]
Cool! I like it.

**Comment by seankross** [comment]
Thanks for all of this work John.

**Comment by muschellij2** [comment]
Any thoughts on pulling this?

**Comment by muschellij2** [comment]
Thoughts?

**Comment by seankross** [comment]
@muschellij2, are you confident that all of this works nicely?

**Comment by seankross** [comment]
@muschellij2 let me rephrase, if you're confident that everything here works then I can push these changes to CRAN.

**Comment by muschellij2** [comment]
Can you test locally on your machine or get your creds on Travis?

John

**Comment by seankross** [comment]
@muschellij2 running the tests locally I get the following error: 

```
Unknown encoder 'libfdk_aac'
```

I followed the most sane looking advice online and it didn't fix anything. Do you have any tips?

Session info:

```
R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.6
```

**Comment by muschellij2** [comment]
Mac OSX brew install should be with `--enable-gpl --enable-nonfree --enable-libfdk-aac` as per https://ffmpeg.org/ffmpeg-codecs.html#libfdk_005faac

**Comment by seankross** [comment]
@muschellij2 The tests are passing for me locally! I am comfortable to merge this into `dev` if you are.

**Comment by muschellij2** [comment]
I am - I think these should reflect the corrections I have.  I think we
should figure out how to authorize the travis app for checks (within
reason).

Best,
John

**Comment by seankross** [comment]
@muschellij2 Can you get this to pass on `devtools::check_rhub()`? So far I cannot.

**Comment by seankross** [comment]
Nvm, please disregard.
